lear-db/Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM postgres:11
 COPY lear_db.sql /docker-entrypoint-initdb.d/
 EXPOSE 5432
+RUN apt update
+RUN apt install sudo
```

Comment:
not sure why these are added

Reply:
in order to shell into a postgres container you need to have sudo installed on it, it just got annoying to do it every time so we put it into the build